es-vue-base/src/lib-components/EsFileInput.vue

```diff
@@ -148,6 +148,20 @@ export default {
         },
     },
     methods: {
+        removeSpaceFromFileNames(files) {
+            const newFiles = files.map((file) => {
+            // Create a new Blob object which is a clone of the file
+                const blob = new Blob([file], { type: file.type });
+
+                // Return a new File object with the modified name
+                return new File(
+                    [blob],
```

Comment:
I *think* you can pass the file in here i.e. `[file]`, to create a new `File` instance with the changed name.

Reply:
Ahh that is smart, I will try that out. Thank you Tom!